src/main/java/teammates/ui/pagedata/InstructorCourseEditPageData.java

```diff
@@ -225,6 +227,14 @@ private ElementTag createCancelEditInstructorButton(int panelIndex, boolean isDi
                                  isDisabled);
     }
 
+    private ElementTag createCancelAddInstructorButton() {
+        String buttonContent = "<span class=\"glyphicon glyphicon-remove\"></span> Cancel";
+        String buttonId = "newInstrCancelLink";
```

Comment:
umm maybe you want to change the id? this one sounds like as if there is a link at this button and its gonna direct to another page...

Reply:
I agree the id is a little confusing. However I realized all the other buttons (including the ones which involve cancelling) names it with a `Link` at the end too, so I followed this naming convention.

I'm concerned if naming this differently would affect how the other ids are interpreted.